addon-test-support/@ember/test-helpers/dom/fire-event.js

```diff
@@ -194,8 +195,23 @@ function buildKeyboardEvent(type, options = {}) {
 }
 
 // eslint-disable-next-line require-jsdoc
-function buildFileEvent(type, element, files = []) {
+function buildFileEvent(type, element, options = {}) {
   let event = buildBasicEvent(type);
+  let files;
+  if (Array.isArray(options)) {
+    deprecate(
+      'Passing the `options` param as an array to `triggerEvent` for file inputs is deprecated. Please pass an object with a key `files` containing the array instead.',
+      false,
+      {
+        id: 'ember-test-helpers.trigger-event.options-blob-array',
+        until: '0.8.0',
```

Comment:
Not sure what the process here is so I just went with the next major version

Reply:
Yep, this seems good.